docs/xgc.md

```diff
@@ -124,25 +124,25 @@ Options that change the behavior of the garbage collector.
 
         -Xgc:dnssExpectedTimeRatioMaximum=<value>
 
-: | Setting       | Value          | Default               |
-  |---------------|----------------|-----------------------|
-  | `<value>`     | [percentage]   | 5                     |
+: | Setting       | Value          | Default                       |
+  |---------------|----------------|-------------------------------|
+  | `<value>`     | [percentage]   | 5 for gencon, 5 for balanced  |
 
-: The maximum amount of time spent on garbage collection of the nursery area, expressed as a percentage of the overall time for the last three GC intervals.
+: The maximum percentage of time spent in local garbage collection pauses. For the `gencon` policy, this refers to the amount of time spent on the nursery area of the heap (scavenge operation). For the `balanced` policy, this refers to the amount of time spent on the eden regions of the heap (PGC operation).
 
-: This option applies only to the `gencon` GC policy.
+: This option applies only to the `gencon` and `balanced` GC policies.
 
 ### `dnssExpectedTimeRatioMinimum`
 
         -Xgc:dnssExpectedTimeRatioMinimum=<value>
 
-: | Setting       | Value          | Default               |
-  |---------------|----------------|-----------------------|
-  | `<value>`     | [percentage]   | 1                     |
+: | Setting       | Value          | Default                      |
+  |---------------|----------------|------------------------------|
+  | `<value>`     | [percentage]   | 1 for gencon, 2 for balanced |
 
-: The minimum amount of time spent on garbage collection of the nursery area, expressed as a percentage of the overall time for the last three GC intervals.
+: The minimum percentage of time spent in local garbage collection pauses. For the `gencon` policy, this refers to the amount of time spent on the nursery area of the heap (scavenge operation). For the `balanced` policy, this refers to the amount of time spent on the eden regions of the heap (PGC operation). 
```

Comment:
we may want to slightly correct the wording for PGC (since it does not operate only on eden regions, although they are typically the majority)

Reply:
For example:

For the `gencon` policy, this refers to the amount of time spent in Scavenge operation (on the nursery area of the heap). For the `balanced` policy, this refers to the amount of time spent in PGC operations (mostly on the eden regions, but also other regions for de-fragmentation purposes)